src/main/java/uk/gov/justice/digital/hmpps/cmd/api/config/WebClientConfiguration.kt

```diff
@@ -49,25 +44,35 @@ class WebClientConfiguration(
     authorizedClientRepository: OAuth2AuthorizedClientRepository,
     builder: WebClient.Builder,
   ): WebClient = builder.authorisedWebClient(
-    authorizedClientManagerRequestScope(clientRegistrationRepository, authorizedClientRepository),
-    registrationId = "elite2-api",
+    authorizedClientManager = authorizedClientManagerRequestScope(clientRegistrationRepository, authorizedClientRepository),
+    registrationId = "prison-api",
     url = csrRootUri,
-    timeout,
+    timeout = timeout,
   )
 
   @Bean
-  fun elite2WebClientAppScope(
-    @Qualifier("authorizedClientManagerAppScope") authorizedClientManager: OAuth2AuthorizedClientManager,
+  fun prisonWebClientAppScope(
+    @Qualifier("authorizedClientManager") authorizedClientManager: OAuth2AuthorizedClientManager,
     builder: WebClient.Builder,
   ): WebClient =
-    builder.authorisedWebClient(authorizedClientManager, registrationId = "elite2-api", url = elite2ApiRootUri, timeout)
+    builder.authorisedWebClient(
+      authorizedClientManager = authorizedClientManager,
+      registrationId = "prison-api",
+      url = prisonApiRootUri,
+      timeout = timeout,
+    )
 
   @Bean
   fun csrAPIWebClientAppScope(
-    @Qualifier("authorizedClientManagerAppScope") authorizedClientManager: OAuth2AuthorizedClientManager,
+    @Qualifier("authorizedClientManager") authorizedClientManager: OAuth2AuthorizedClientManager,
     builder: WebClient.Builder,
   ): WebClient =
-    builder.authorisedWebClient(authorizedClientManager, registrationId = "elite2-api", url = csrRootUri, timeout)
+    builder.authorisedWebClient(
+      authorizedClientManager = authorizedClientManager,
+      registrationId = "prison-api",
+      url = csrRootUri,
+      timeout = timeout,
+    )
 
   private fun authorizedClientManagerRequestScope(
```

Comment:
I was thinking about moving this to the kotlin lib too - seen it used in a couple of places now. Though maybe not as I thought we had been encouraged to move away from checking users in tokens.